autofit/database/aggregator/scrape.py

```diff
@@ -11,6 +12,7 @@
 from ..sqlalchemy_ import sa
```

Comment:
Is there a reason to use relative imports here?

Reply:
No mostly likely pycharm put them there automatically